docs/js/files/modules_FormModel.js.html

```diff
@@ -161,6 +161,12 @@ <h1 class="file-heading">File: modules/FormModel.js</h1>
     mapping: undefined,
 
     /**
+     * @property models
+     * @type Object
+     **/
+    models: undefined,
```

Comment:
Okay, it looks like the it falls back on existing tracked models when you update the mappings. The only thing you can't do is call `this.setMapping(...)` and hope it immediately copies the values. You'll have to do a `this.pull()` afterwards